app/views/posts/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<div>
+ <h3>Create New Post</h3>
+  <%= form_with(model:  @post, url: user_posts_path, local: true) do |form| %>
+    <%= form.label :title %>
+    <%= form.text_field :title, name: 'post[title]' %>
+    <%= form.label :text %>
+    <%= form.text_area :text, name: 'post[text]' %>
+    <%= form.submit 'Create Post' %>
+  <% end %>
+</div>
```

Comment:
- [ ] Good job so  far 👏  But you should also create a form in the **views/comments/new.html.erb** file which with input for creating comments also please create a route to get this form in order to make your project as expected 👍